README.md

```diff
@@ -1 +1,103 @@
-# SSVerticalPanorama
\ No newline at end of file
+# SSVerticalPanoramaImage
+## Capture stunning vertical panorama images with ease and preview them instantly on the built-in screen.
+[![CI Status](https://img.shields.io/travis/SimformSolutionsPvtLtd/SSVerticalPanoramaImage.svg?style=flat)](https://travis-ci.org/SimformSolutionsPvtLtd/SSVerticalPanoramaImage)
+[![Version](https://img.shields.io/cocoapods/v/SSVerticalPanoramaImage.svg?style=flat)](https://cocoapods.org/pods/SSVerticalPanoramaImage)
+[![License](https://img.shields.io/cocoapods/l/SSVerticalPanoramaImage.svg?style=flat)](https://cocoapods.org/pods/SSVerticalPanoramaImage)
+[![Platform](https://img.shields.io/cocoapods/p/SSVerticalPanoramaImage.svg?style=flat)](https://cocoapods.org/pods/SSVerticalPanoramaImage)
+
+SSVerticalPanoramaImage provides a feature to capture a vertical panorama image and also provides an option to preview it on the built-in screen.
+
+# Features
+
+- Capture vertical panorama image
+- Swipe the direction of capture from top to bottom and bottom to top
+- Inbuilt flash and zoom functionality
+
+# 🎬 Preview
+Under development...
+
+# Installation
+
+### CocoaPods
+
+[CocoaPods][cocoapods-link] is a dependency manager for Cocoa projects. You can install it with the following command:
+
+```bash
+$ gem install cocoapods
+```
+
+To integrate SSVerticalPanoramaImage into your Xcode project using CocoaPods, specify it in your `Podfile`:
+
+```ruby
+platform :ios, '15.0'
+use_frameworks!
+
+target '<Your Target Name>' do
+    pod 'SSVerticalPanoramaImage'
+end
+```
+
+Then, run the following command:
+
+```bash
+$ pod install
+```
+
+# How to use:
+
+ 1. Create an instance of SSVerticalPanorama and call `openSSVerticalPanoramaCam`.
+ 
+ ```swift
+@IBAction func onClickOfOpenCamera(_ sender: UIButton) {
+    guard let navController = navigationController else { return}
+    let ssVerticalPano = SSVerticalPanorama()
+    ssVerticalPano.delegate = self
+    ssVerticalPano.openSSVerticalPanoramaCam(navController: navController)
+}
+```
+ 2. Confirm the `SSVerticalPanoDelegate` delegate to receive the captured image.
+ ```swift
+ //MARK: - SSVerticalPanoDelegate
+extension ViewController: SSVerticalPanoDelegate {
+    func onImageCaptured(image: UIImage) {
+        // Use image
+    }
+}
+ ```
+# How to set in-built preview scereen :
+* Set `showPreviewScreen` to `true` or `false` to enable or disable in-built preview screen.
+```swift
+ssVerticalPano.showPreviewScreen = true
+```
+
+## Find this library useful? ❤️
+Support it by joining __[stargazers]__ for this repository.⭐
+
+## 🤝 How to Contribute
+
+Whether you're helping us fix bugs, improve the docs, or a feature request, we'd love to have you! 💪
+Check out our __[Contributing Guide]__ for ideas on contributing.
+
+# Check our other Libraries:
+* Also, Check our other libraries - [SimformSolutionsPvtLtd][SimformSolutionsPvtLtd]
```

Comment:
Use this repo link for ref. 
https://github.com/SimformSolutionsPvtLtd/Awesome-Mobile-Libraries